WalletWasabi/Wallets/IWallet.cs

```diff
@@ -10,7 +10,7 @@ namespace WalletWasabi.Wallets;
 
 public interface IWallet
 {
-	string WalletName { get; }
+	int Id { get; }
```

Comment:
Can we keep both to maintain compatibility with BTCPay? 

@Kukks can you take a look?

Reply:
Does it have to be a number type? I use it to match a wallet to a btcpay store

Reply:
![image](https://github.com/zkSNACKs/WalletWasabi/assets/9844978/e15cc2dd-a0a1-4502-a544-86c876d909d0)